modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -2322,15 +2489,19 @@ This command does not take any options, except for the default ones: `--help`, `
 ----
 > cloudhub region list [options]
 ----
-This command lists all supported regions. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all supported regions. 
+
+// CLI DEFAULTS
+include::partial$cli-default-options.adoc[tag=CLIdefaultOptions]
 
 === cloudhub runtime list
 ----
 > cloudhub runtime list [options]
 ----
-This command lists all supported runtimes. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command lists all supported runtimes.
```

Comment:
Are we talking about Mule runtime engines? Or some other runtime. Only use "runtimes" alone if Mule is understood from the context.